src/ViewRendererBundle/DependencyInjection/Configuration.php

```diff
@@ -19,8 +19,8 @@ public function getConfigTreeBuilder(): TreeBuilder
         $root
             ->children()
                 ->scalarNode('css_dir')
-                    ->defaultValue(PATH_USER_CMS_PUBLIC.'/outbox/static/less')
-                    ->info('Directory path to where the less compiler caches work and the output files are stored.')
+                    ->defaultValue('/chameleon/outbox/static/less')
+                    ->info('Directory path to where the less compiler cache works and the output files are stored. Relative to the web root directory. All the files here are deleted on cache clear.')
```

Comment:
Suggestion:

"Path used as working and output directory for the less compiler, relative to the web root directory. CAUTION: All files in this directory are deleted on cache clear, so be sure not to use this directory for any other purpose."

Reply:
ok